applications/FluidDynamicsApplication/custom_utilities/mass_conservation_utility.h

```diff
@@ -194,27 +234,34 @@ class KRATOS_API(FLUID_DYNAMICS_APPLICATION) MassConservationCheckProcess : publ
     ///@{
 
     // Reference to the model part
-    const ModelPart& mrModelPart;
+    ModelPart& mrModelPart;
 
-    // Process parameters
-    int mCorrectionFreq = 1;
-    bool mWriteToLogFile = true;
-    bool mPerformCorrections = true;
-    std::string mLogFileName = "mass_conservation.log";
+    // Level of verbosity
+    int mEchoLevel;
 
     // Inital volume with negative distance field ("water" volume)
     double mInitialNegativeVolume = -1.0;
     // Inital volume with positive distance field ("air" volume)
     double mInitialPositiveVolume = -1.0;
 
+    //Name of time variable
+    std::string mTimeVariableName;
```

Comment:
Why don't we save a const-ref or pointer to the variable to avoid the Kratos Components Get each time we require the variable.

Reply:
done